src/libcore/tests/char.rs

```diff
@@ -32,7 +32,6 @@ fn test_convert() {
 #[test]
 fn test_from_str() {
     assert_eq!(char::from_str("a").unwrap(), 'a');
-    assert_eq!(char::try_from("a").unwrap(), 'a');
```

Comment:
Why was this test removed? Shouldn't it succeed?

Reply:
This was because of the `FromStr` --> `TryFrom` impl being removed. The test fails without it.